### ISSUE TITLE
Does not run under Django 4.0.x

### ISSUE THREAD

**Issue by nephros** [CODeRUS/django-test#3]
```
  File "/home/www/test/htdocs/pm2/example/demo/apps.py", line 2, in <module>
    from django.utils.translation import ugettext_lazy as _
ImportError: cannot import name 'ugettext_lazy' from 'django.utils.translation' (/usr/lib/python3.9/site-packages/django/utils/translation/__init__.py)

```

Reason: `ugettext_lazy` is deprecated in newer django versions.
Apparently can be fixed by using `gettext_lazy`  but haven't tried.

See also:

 -  https://stackoverflow.com/questions/70656495/importerror-cannot-import-name-ugettext-lazy
 - https://forum.djangoproject.com/t/importerror-cannot-import-name-ugettext-lazy-from-django-utils-translation/10943/3

**Comment by CODeRUS** [comment]
thats true, patchmanager catalog is using Django 2.1.5 version
if you want to keep it running on latest version you have to modify code.

**Comment by nephros** [comment]
Yeah, it's not immediately necessary to change it, just wanted to document what *could* be done.

Django 3.x works too btw.
